recipes-samples/images/rpb-qt5-image.bb

```diff
@@ -0,0 +1,46 @@
+SUMMARY = "Basic QT5 EGLFS image"
+
+IMAGE_FEATURES += " package-management debug-tweaks ssh-server-openssh hwcodecs tools-debug"
+
+LICENSE = "MIT"
+
+inherit core-image distro_features_check extrausers
+
+# let's make sure we have a good image..
+REQUIRED_DISTRO_FEATURES = "pam systemd opengl"
+
+# by default and without x11-base in IMAGE_FEATURES we default to multi-user.target
+SYSTEMD_DEFAULT_TARGET = "multi-user.target"
+
+CORE_IMAGE_BASE_INSTALL += " \
+    96boards-tools \
+    alsa-utils-aplay \
+    networkmanager \
+    networkmanager-nmtui \
+    coreutils \
+    gps-utils \
+    gpsd \
+    gptfdisk \
+    ${@bb.utils.contains("DISTRO_FEATURES", "x11", "gstreamer1.0-plugins-bad-meta", "", d)} \
```

Comment:
hmm. i found the issue, it is a bug upstream, but in meta-qcom ;-) it's coming from 
https://github.com/ndechesne/meta-qcom/blob/master/recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_%25.bbappend

i will fix it. then we can leave gstreamer1.0-plugin-bad-meta in the image.. thanks for spotting it..